src/shared/components/mutationTable/column/ascnCopyNumber/ASCNCopyNumberElement.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import DefaultTooltip from "public-lib/components/defaultTooltip/DefaultTooltip";
+import SampleManager from "pages/patientView/SampleManager";
+
+const ASCNCallTable:{[key:string]:string} = {
+    "no WGD,0,0":"Homiodel",
+    "no WGD,1,0":"Hetloss",
+    "no WGD,2,0":"CNLOH",
+    "no WGD,3,0":"CNLOH & Gain",
+    "no WGD,4,0":"CNLOH & Gain",
+    "no WGD,5,0":"Amp (LOH)",
+    "no WGD,6,0":"Amp (LOH)",
+    "no WGD,1,1":"Diploid",
+    "no WGD,2,1":"Gain",
```

Comment:
And in the following strings, you should move them to constants as many as possible just in case they will be updated later on, then you don't need to modify everywhere